galsim/scene.py

```diff
@@ -86,46 +92,58 @@ class COSMOSCatalog(object):
-                        (in arcsec).  [default: 0, meaning no limit]
-    @param min_flux     Exclude galaxies whose fitted flux is smaller than this value.  
-                        [default: 0, meaning no limit]
-    @param max_flux     Exclude galaxies whose fitted flux is larger than this value.  
-                        [default: 0, meaning no limit]
+    @param file_name        The file containing the catalog. [default: None, which will look for the
+                            F814W<25.2 COSMOS catalog in $PREFIX/share/galsim.  It will raise an
+                            exception if the catalog is not there telling you to run
+                            galsim_download_cosmos.]
+    @param sample           A keyword argument that can be used to specify the sample to use, i.e.,
+                            "23.5" or "25.2".  At most one of `file_name` and `sample` should be
+                            specified.
+                            [default: None, which results in the same default as `file_name=None`.]
+    @param image_dir        Keyword that is only used for real galaxies, not parametric ones, to
+                            specify the directory of the image files.
+                            If a string containing no `/`, it is the relative path from the location
+                            of the catalog file to the directory containing the galaxy/PDF images.
+                            If a path (a string containing `/`), it is the full path to the
+                            directory containing the galaxy/PDF images. [default: None]
+    @param dir              The directory of catalog file. [default: None]
+    @param preload          Keyword that is only used for real galaxies, not parametric ones, to
+                            choose whether to preload the header information.  If `preload=True`,
+                            the bulk of the I/O time is in the constructor.  If `preload=False`,
+                            there is approximately the same total I/O time (assuming you eventually
+                            use most of the image files referenced in the catalog), but it is spread
+                            over the various calls to getGal() and getPSF().  [default: False]
+    @param noise_dir        Keyword that is only used for real galaxies, not parametric ones.
+                            The directory of the noise files if different from the directory of the 
+                            image files.  [default: image_dir]
+    @param use_real         Enable the use of realistic galaxies?  [default: True]
+                            If this parameter is False, then `makeGalaxy(gal_type='real')` will
+                            not be allowed.
+    @param exclusion_level  Level of cuts to make on the galaxies based on the quality of postage
+                            stamp definition and/or parametric fit quality.  Options:
+                            "none": No cuts.
```

Comment:
Well - those cuts were really minimal because we were trying to get as close as we could to a flux-limited sample.  But I suppose technically we should mention them and point to their description.  Done.